packages/Takeover/src/tokens.js

```diff
@@ -0,0 +1 @@
+export const animationDuration = 700;
```

Comment:
I like this, for package local const variable. cool!

Reply:
Me too.  Would it be more appropriate in `/helpers` though?

Reply:
I think is clearer having their own tokens files. Helpers tend to be more functions/methods specifics for the logic, and this is for styling and constant variables.

Reply:
I mean `/src/helpers/tokens.js`, just so the `/src` folder stays a bit tidier.